documentation/book/assembly-maintenance-time-windows.adoc

```diff
@@ -0,0 +1,32 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-maintenance-time-windows-{context}']
+
+= Maintenance time windows for rolling updates
+
+Maintenance time windows allow you to schedule rolling updates of your Kafka and Zookeeper clusters to start at a convenient time.
+During a rolling update, pod instances in a cluster are updated incrementally, with no downtime.
+
+You configure a maintenance time window by entering an array of strings containing one or more {CronExpressions} in the `Kafka.spec.maintenanceTimeWindows` property.
+The cron expressions must be expressed in UTC.
+The following example shows a typical maintenance time window:
+
+[source,yaml,subs="attributes+"]
+----
+# ...
+maintenanceTimeWindows:
+  - "* * 8-10 * * ?"
+  - "* * 14-15 * * ?"
+# ...
+----
+
+NOTE: Currently, maintenance time windows for rolling updates are only respected by the Certificate Authority (CA) certificate renewal process.
+Other processes—for example, a rolling update triggered by configuring a Kafka cluster—do not respect maintenance time windows.
```

Comment:
Please use em dashes (denoted by two hyphens -- in Asciidoctor). 

--for example, a rolling update triggered by configuring a Kafka cluster--